src/Pipe.rei

```diff
@@ -0,0 +1,13 @@
+// Borrowed from ReactMini - the "pipe" (our term) is a mechanism for
```

Comment:
Cool idea to factor this into something more explicit!

It might be worth mentioning `handedOffInstance` here - ie, "Borrowed from ReactMini's handedOffInstance".

Reply:
Good idea. Added.